### PR TITLE
Update chromium sample and clarify sample READMEs

### PR DESCRIPTION
**Pull request by aaronliu0130**
A chromium sample has been updated to a version that uses C++17 stuff

READMEs of samples have been clarified regarding licensing

Two licenses have also been updated

Our setup.py now includes a BSD classifier for PyPI

### CODE REVIEW

File: samples/protobuf-sample/README.md [hunk 1]
```diff
@@ -1,3 +1 @@
-# VLC sample
-
-code taken for regression testing from https://github.com/protocolbuffers/protobuf
+Code under src/ taken from https://github.com/protocolbuffers/protobuf. Rights except redistribution reserved to Google. See LICENSE.
```
Comment (jayvdb) [inline samples/protobuf-sample/README.md]: "Rights except redistribution reserved to Google." is incorrect.  The license is BSD-3-Clause

Comment (aaronliu0130) [reply samples/protobuf-sample/README.md]: Okay, I'll go ahead and correct every instance.

File: samples/v8-sample/README.md [hunk 2]
```diff
@@ -1,3 +1 @@
-# V8 sample
-
-code taken for regression testing from https://github.com/v8/v8
+Code under src/ taken from https://github.com/v8/v8. Rights except redistribution reserved to the V8 project authors. See LICENSE.
```
Comment (jayvdb) [inline samples/v8-sample/README.md]: "Rights except redistribution reserved to the V8 project authors." is incorrect. It is BSD-3-Clause

Comment (aaronliu0130) [reply samples/v8-sample/README.md]: Yeah, I was mistaken. I also made that mistake on two others. I'll correct it soon

**Review by jayvdb — APPROVED** [review]

**Pull request merged**